compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -70,7 +70,7 @@ hir_analysis_coercion_between_struct_same_note = expected coercion between the s
 hir_analysis_coercion_between_struct_single_note = expected a single field to be coerced, none found
 
 hir_analysis_const_bound_for_non_const_trait =
-    ~const can only be applied to `#[const_trait]` traits
+    {$modifier} can only be applied to `#[const_trait]` traits
```

Comment:
```suggestion
    `{$modifier}` can only be applied to `#[const_trait]` traits
```

Reply:
`modifier` is already quoted by the pre-existing `Display` impl, I can try to rewrite  it not to do that.